BatchApex/BatchApex.cls

```diff
@@ -0,0 +1,134 @@
+                    StageName = 'Closed Won'
+                AND
+                    CloseDate = TODAY
+            ]);
+    }
+    
+    /**
+     * This will sum up all the closed won opportunities of a perticular account per day
+     * and will send a mail to the primary contact
+     */
+    global void execute(
+        Database.BatchableContext batchableContext,
+        List<Opportunity> opportunityList) {
+        for (Opportunity opportunityInstance : opportunityList) {
+            if (accountOpportunityMap.containsKey(opportunityInstance.Account.Primary_Email__c)) {
+                WrapperClass wrapperInstance = 
+                    accountOpportunityMap.get(opportunityInstance.Account.Primary_Email__c);
+                
+                // Add the opportunity in the opportunity list in the wrapper class
+                wrapperInstance.opportunityList.add(opportunityInstance);
+                
+                // Add the amount of that opportunity which is inserted in the opportunity list 
+                // in the sum variable in the wrapper class
+                wrapperInstance.amountSum += opportunityInstance.Amount;
+                totalOpportunityAmount += opportunityInstance.Amount;
+                accountOpportunityMap.put(
+                    opportunityInstance.Account.Primary_Email__c, wrapperInstance);
+            } else {
+                wrapperClass wrapperInstance = new wrapperClass();
+                wrapperInstance.opportunityList.add(opportunityInstance);
+                wrapperInstance.amountSum = opportunityInstance.Amount;
+                totalOpportunityAmount = opportunityInstance.Amount;
+                accountOpportunityMap.put
+                    (opportunityInstance.Account.Primary_Email__c, 
+                    wrapperInstance
+                );
+            }
+        }
+    }
+    
+    /**
+     * This is an execute block of schedulable.
+     */
+    global void execute(SchedulableContext schedulableContext) {
+        
+    }
+    
+    /**
+     * The finish block to send mail to primary contact of all opportunities 
+     * with the sum of their amount.
+     */
+    global void finish(Database.BatchableContext batchableContext) {
+        List<Messaging.SingleEmailMessage> mailInstance = new List<Messaging.SingleEmailMessage>();
+        List<String> sendToList = new List<String>();
+        for (String emailInstance : accountOpportunityMap.keySet()) {
+            Messaging.SingleEmailMessage individualMail = new Messaging.SingleEmailMessage();
+            WrapperClass wrapperInstance = accountOpportunityMap.get(emailInstance);
+            
+            // Recipient email id
+            sendToList.add(emailInstance);
+            individualMail.setToAddresses(sendToList);
+            individualMail.setSenderDisplayName('SalesForce.com');
+            individualMail.setSubject('List Of Opportunities and Amount');
+            String body = 'Hello ' + IndividualMail;
```

Comment:
This mail should be sent to  primary contact? here u are sending it to Acount's Email. This is again wrong. Please get clear with your requirements first. Think wht u need to implement and then start with coding. Before Coding I want u to discuss with me .I want to know whether u have understood the requirement or not and wht are you actually going to implement. Here the name also should be of Primary Contact